bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/tcc/TccNames.java

```diff
@@ -23,6 +23,7 @@ private TccNames() {
     public static final String AZEL = "AzEl";
     public static final String BASE = "Base";
 //    public static final String BASE = "sciencetarget";
+    public static final String BAGSTARGET = "bagsTarget";
```

Comment:
Hmmm... we haven't, and I'm not sure what the best way is to go about doing that.
Anyone know who I should talk to?